src/tmplits/tmplits-table/module.js

```diff
@@ -27,7 +27,7 @@ export function TmplitTable(context, args) {
         attr
     } = util.cleanArgs(_args)
 
-    classList = classList.concat(['tmplit-table'])
+    classList.push('tmplit-table');
```

Comment:
```suggestion
    classList.push('tmplit-table')
```
Style nitpick since the rest of the JS here doesn't use semi-colons